src/msg/msg_types.cc

```diff
@@ -290,3 +290,22 @@ void entity_addrvec_t::generate_test_instances(list<entity_addrvec_t*>& ls)
   ls.back()->v.push_back(entity_addr_t());
   ls.back()->v.push_back(entity_addr_t());
 }
+
+const std::string ip_only_to_str() 
```

Comment:
i mean, `std::string ip_only_to_str() const`. as this method is not changing `in4_addr` IIUC.

Reply:
@tchaikov 
updated it, thank you